src/App.js

```diff
@@ -13,7 +13,8 @@ class App extends Component {
     this.state = {
       bigData: {},
       cardsArray: [],
-      view: 'initial'
+      view: 'initial',
+      comparedData: {}
```

Comment:
We've added an object in order to create and Object.assign three new cards into.